### PR TITLE
[enhancement] Add per-section freshness metadata to Coinbase account snapshots

### LINKED ISSUE #644: [enhancement] Add per-section freshness metadata to Coinbase account snapshots
## Why / Context
`CoinbaseAccountManager.snapshot()` currently returns mixed payloads without explicit freshness metadata per section. During incidents, operators can’t easily tell whether a section is fresh, stale, or failed without parsing logs.

## Scope
**In scope**
- Add per-section freshness metadata (for example: `status`, `fetched_at`, and optional `error_code`) in account snapshot output.
- Preserve existing top-level snapshot keys to avoid breaking consumers.
- Include INTX/CFM sections in the standardized freshness shape.

**Out of scope**
- Coinbase API contract changes.
- Replacing existing snapshot event transport.

Constraints:
- Keep changes small/mergeable.
- Deterministic tests only.
- Avoid touching unrelated generated artifacts unless required.

## Acceptance Criteria (required)
- [ ] Snapshot payload includes explicit freshness/status metadata for major sections.
- [ ] Existing snapshot keys remain backward compatible.
- [ ] Temporary fetch errors produce deterministic metadata instead of only free-form strings.
- [ ] Unit tests validate both successful and partial-failure metadata outputs.

## Implementation Notes / Pointers
**Likely files / modules:**
- `src/gpt_trader/features/brokerages/coinbase/account_manager.py`
- `tests/unit/gpt_trader/features/brokerages/coinbase/test_coinbase_account_manager_snapshot.py`
- `tests/unit/gpt_trader/features/live_trade/telemetry/test_account_telemetry.py`

**Related tests:**
- `tests/unit/gpt_trader/features/brokerages/coinbase/test_coinbase_account_manager_snapshot.py`
- `tests/unit/gpt_trader/features/live_trade/telemetry/test_account_telemetry.py`

**Edge cases to handle:**
- INTX unavailable vs temporary INTX failure.
- Section succeeds with empty data vs section fails.
- Ensure no sensitive values appear in metadata fields.

## Commands (local)
- `make fmt`
- `make lint`
- `uv run pytest -q tests/unit/gpt_trader/features/brokerages/coinbase/test_coinbase_account_manager_snapshot.py tests/unit/gpt_trader/features/live_trade/telemetry/test_account_telemetry.py`

## PR Requirements
- PR title should match the issue.
- PR body must include: `Fixes #<issue-number>`
- CI must be green; if Agent Artifacts Freshness fails, run `uv run agent-regenerate`, then commit `var/agents/...` updates.

## Codex-Ready Checklist (for the issue creator)
- [x] Clear acceptance criteria
- [x] At least one file pointer
- [x] Commands included
- [x] No ambiguous “do the right thing” language

### PR DESCRIPTION
**Pull request by Solders-Girdles**
Fixes #644

(Automated PR created by OpenClaw PR finisher)
Branch: codex/644-enhancement-add-per-section-freshness-me
Commit: c6d19c40b77562a04dff0a0cd8ee5a11c65c45fe

### CODE REVIEW

File: src/gpt_trader/features/brokerages/coinbase/account_manager.py [hunk 1]
```diff
@@ -122,6 +147,95 @@ def _snapshot_error_payload(error: Exception) -> dict[str, Any]:
...
+                        ),
+                        refreshed_uuid,
+                    )
+            return (
+                [],
+                self._freshness_entry(
+                    self._FRESHNESS_ERROR,
+                    error_code=self._error_code_from_exception(error),
+                ),
+                portfolio_uuid,
```
[the remark this answers is not in the document]
Comment (Solders-Girdles) [reply src/gpt_trader/features/brokerages/coinbase/account_manager.py]: Valid finding — fixed in e7807aa0.

Three changes:

1. `_fetch_intx_balances_with_retry` now returns `None` (instead of the original `portfolio_uuid`) when refresh returns no portfolio, so the stale UUID doesn't leak back to callers.
2. `snapshot()` uses `target_uuid = resolved_uuid` instead of `resolved_uuid or intx_portfolio_uuid`, removing the fallback that would re-introduce the stale UUID.
3. Added `test_stale_uuid_cleared_when_refresh_returns_none` covering this exact scenario.

File: src/gpt_trader/features/brokerages/coinbase/account_manager.py [hunk 2]
```diff
@@ -39,60 +45,72 @@ def snapshot(self) -> dict[str, Any]:
...
-            snapshot_data["intx_collateral"] = {}
+            result, metadata = self._execute_snapshot_probe(key, probe_name)
+            snapshot_data[key] = result
+            freshness_data[key] = metadata
+
+        intx_available = self.broker.supports_intx()
+        snapshot_data["intx_available"] = intx_available
+        freshness_data["intx_available"] = self._freshness_entry(
+            self._FRESHNESS_FRESH if intx_available else self._FRESHNESS_UNAVAILABLE,
+            error_code=None if intx_available else "INTX_NOT_SUPPORTED",
```
[the remark this answers is not in the document]
Comment (Solders-Girdles) [reply src/gpt_trader/features/brokerages/coinbase/account_manager.py]: Valid finding — fixed in e7807aa0.

`_record_intx_fallback` now updates `freshness_data["intx_available"]` alongside the section-level freshness entries, so when `intx_available` is flipped to `False` the freshness metadata stays consistent. The new `test_stale_uuid_cleared_when_refresh_returns_none` test asserts `freshness(snapshot, "intx_available")["status"] == "unavailable"` to cover this.
